archivebox/extractors/media.py

```diff
@@ -26,7 +26,9 @@ def should_save_media(link: Link, out_dir: Optional[Path]=None, overwrite: Optio
         return False
 
     out_dir = out_dir or Path(link.link_dir)
-    if not overwrite and (out_dir / 'media').exists():
+    media_dir = out_dir / 'media'
+    media_dir_has_files = media_dir.exists() and any(media_dir.iterdir())
```

Comment:
```suggestion
    media_dir_has_files = media_dir.exists() and any(filename for filename in media_dir.iterdir() if not filename.startswith('.'))
```
Should we exclude `.DS_Store` files and other hidden temp files here?